src/js/disability-benefits/526EZ/config/form.js

```diff
@@ -122,50 +130,56 @@ const formConfig = {
   subTitle: 'Form 21-526EZ',
   // getHelp: GetFormHelp, // TODO: May need updated form help content
   chapters: {
-    reviewVeteranDetails: {
-      title: (isReviewPage) => {
-        const baseString = 'Review Veteran Details';
-        if (isReviewPage) {
-          return baseString.replace('Review ', '');
-        }
-        return baseString;
-      },
+    veteranDetails: {
+      title: (isReviewPage) => `${isReviewPage ? 'Review ' : ''}Veteran Details`,
       pages: {
-        veteranInformation: createVerifiedVeteranInfoPage(fullSchema526EZ),
-        specialCircumstances: { // TODO: create page file and reuse 
-          title: 'Special Circumstances',
-          path: 'review-veteran-details/special-circumstances',
-          depends: formData => formData.prefilled,
+        veteranInformation: {
+          title: 'Veteran Details', // TODO: Figure out if this is even necessary
+          path: 'veteran-information',
           uiSchema: {
-            'ui:description': specialCircumstancesDescription,
-            'view:suicidal': {
-              'ui:title': 'In crisis or thinking of suicide?'
+            'ui:field': ReviewCardField,
+            'ui:title': 'Veteran information',
+            'ui:options': {
+              viewComponent: veteranInformationViewField
             },
-            'view:homeless': {
-              'ui:title': 'Homeless or at risk of becoming homeless?'
-            },
-            'view:extremeFinancialHardship': {
-              'ui:title': 'Suffering from extreme financial hardship?'
+            fullName: fullNameUI,
+            socialSecurityNumber: merge({}, ssnUI, {
```

Comment:
Uhhh, just now realizing that there's no place for us to submit this data in the Swagger. Do we actually need ssn? Also applies to a bunch of other veteran info.